debian/control

```diff
@@ -0,0 +1,22 @@
+Source: libpam-policycache
+Priority: extra
+Maintainer: Nikki VonHollen <vonhollen@google.com>
+Build-Depends: debhelper (>= 8.0.0), autotools-dev, libglib2.0-dev,
+ libpam0g-dev, automake, libtool, autogen, libscrypt-dev
+Standards-Version: 3.9.2
+Section: libs
+Homepage: https://github.com/google/libpam-policycache
+
+Package: libpam-policycache
+Section: libs
+Architecture: any
+Depends: ${shlibs:Depends}, ${misc:Depends}, libscrypt0
```

Comment:
Issue pkern@ found that caused libscrypt to link statically: https://bugs.launchpad.net/ubuntu/+source/libscrypt/+bug/1313311

It's safe to remove the dependency and it comes back automatically in Debian testing since it's linked properly.